### DISCUSSION TITLE
Is there anyway to view metrics for all the different HTTP requests that are being sent inside a playwright test?

### DISCUSSION THREAD

**Discussion by ATheCoder** [artilleryio/artillery#2490] [Artillery]
When I visit a webpage in chrome and I open the Network tab I can see a lot of requests being sent. Artillery gives me an overview of the number of requests being sent on a playwright scenario However they are not separated based on URL or Path.

Is there anyway to separate these metrics such as P95 or median based on HTTP endpoint URL when running a playwright scenario?

I wanna know which endpoints are the slowest

**Comment by hassy** [comment]
Not right now @ATheCoder. We're considering extending [tracing](https://www.artillery.io/docs/reference/extensions/publish-metrics/opentelemetry) with details of resource requests but no ETA on that yet.
